EnglishFilter/sections/foreign.txt

```diff
@@ -4355,6 +4355,8 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/90634
+ygosu.com#?#a[href^="https://ygosu.com/banner2/"]:upward(tbody > tr)
```

Comment:
I'd use this `ygosu.com#?#tbody > tr a[href^="https://ygosu.com/banner"]:upward(2)`

Reply:
Did you mean the following changes?
```suggestion
ygosu.com#?#tbody > tr > a[href^="https://ygosu.com/banner"]:upward(2)
```

Reply:
Almost, I've added my variant without `>` - it matches the banner.